src/ipahealthcheck/ipa/certs.py

```diff
@@ -879,3 +879,35 @@ def check(self):
             else:
                 yield Result(self, constants.SUCCESS,
                              key=ca)
+
+
+@registry
+class IPAMainCAExpirationCheck(IPAPlugin):
+
+    @duration
+    def check(self):
+        try:
+            cert = x509.load_certificate_from_file(paths.IPA_CA_CRT)
```

Comment:
This should be load_certificate_list_from_file() otherwise only the first cert will be loaded. We want to check the entire chain.